mmeds/tests/unit/test.py

```diff
@@ -159,5 +138,33 @@ def remove_users(users_added):
             remove_user(fig.TEST_USER_0, testing=testing)
 
 
+def main():
+    tests = sys.argv[1:]
```

Comment:
I'm not really sure what this code is doing, a couple comments might help.

Reply:
It's grabbing the arguments that were passed to the python script. The first argument is the path to the script itself so I'm skipping that one. Since we use click for arguments in the lab it makes sense you haven't seen this but it's very standard. I suppose I could switch to it here but it would be less convenient in this situation I think

Reply:
ah that makes sense.